.github/workflows/start_bake_time.yml

```diff
@@ -0,0 +1,21 @@
+name: Start bake time
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+permissions:
+  checks: 'write'
+  issues: 'write'
+  pull-requests: 'write'
```

Comment:
@opensearch-project/engineering-effectiveness I'm getting an error that the GITHUB_TOKEN doesn't have access to make these changes, do you have thoughts or a recommendations for how to resolve this?

```
    data: {
      message: 'Resource not accessible by integration',
      documentation_url: 'https://docs.github.com/rest/reference/issues#add-labels-to-an-issue'
    }
```

Reply:
It is strange as the there is another workflow that adds labels to issues - no problems https://github.com/opensearch-project/.github/blob/main/.github/workflows/add-untriaged.yml

I'll try more experimentation within this pull request - maybe I am passing the wrong token or maybe action/github-script works differently as a composite action

Reply:
Circling back on this, when a pull request triggers a github action it cannot perform _any_ write operations.  This is a good security measure, and in order to make this functional I had to drop all write actions during this trigger.  Now the check 'fails' on the pull request with the message saying how long it will take.

If we want a cleaner experience a bot will need to execute the actions.